SayTheirNames/Source/Dependency/Deep Linking/DeepLink.swift

```diff
@@ -26,10 +26,9 @@ import UIKit
 
 protocol DeepLink {
     static var details: DeepLinkDetails { get }
-    static func deepLink(fromComponents: [String]?) -> DeepLink?
-    var linkDetails: DeepLinkDetails { get }
+    init?(components: [String]?)
 }
 
-protocol DeepLinkHandle {
-    func handle(deepLink: DeepLink)
+protocol DeepLinkPresentable {
```

Comment:
isn't it more like `DeepLinkPresenter`? "presentable" would be something that can be presented.

Reply:
Done.